cmd/fyne/main.go

```diff
@@ -61,6 +61,7 @@ func loadCommands() {
 		{"install", &installer{}},
 		{"vendor", &vendor{}},
 		{"version", &version{}},
+		{"server", &server{}},
```

Comment:
Nice, but should probably be "serve" to match verb based commands ("package", "install")